dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml

```diff
@@ -0,0 +1,5 @@
+description: Generic Zephyr SPI based SDHC controller
+
+compatible: "zephyr,sdhc-spi-slot"
+
+include: [sdhc.yaml]
```

Comment:
actually, spi device should `include: spi-device.yaml`

Reply:
Changed to `spi-device.yaml`.